### ISSUE TITLE
Any possibility you could possibly keep this updated for pulsar, or we could fork it?

### ISSUE THREAD

**Issue by kaosine** [wysiib/linter-languagetool#41]
I'm from the group running [Pulsar](https://github.com/pulsar-edit) a fork of Atom and working to maintain it. While we're making updates to it, and we wanted to replace the functionality of the default spell checker with something more up to date. It seems plausible that this would be a suitable add-on to have whether it's still maintained by you and contributions come from interested users or if there's no problem with us forking it to use as a core add-on. 

If it helps we also have a discord you can join since we're trying to build up our native add-ons that aren't just backups of the old Atom ones. I'll attach a link to that below (and hope this doesn't seem to out of place making this issue here and that you don't consider it to be spam somehow)

https://discord.gg/7aEbB9dGRT

**Comment by wysiib** [comment]
Hi @kaosine,

as I'm not actively using Atom since quite some time and I am not actively using Pulsar (yet?), I don't see myself maintaining the extension. Feel free to fork and / or take over maintenance of this repository. Currently, I haven't yet decided what will happen to this repository (and my other repositories including Atom extensions) once Atom finally retires.

**Comment by ottok** [comment]
This package is listed at https://web.pulsar-edit.dev/packages/linter-languagetool and I am using it with Pulsar.

**Comment by wysiib** [comment]
@ottok is it working just fine? Is there anything to be done?

As stated above, I am not doing active maintenance but would be willing to accept PRs, etc.

**Comment by ottok** [comment]
Yes, it works. I just started using it, no suggestions so far. Good to know you are open to PRs.

![image](https://github.com/user-attachments/assets/87a7403e-9049-4858-a2c9-b8893cdcb457)

![image](https://github.com/user-attachments/assets/bb7757ad-1ef8-4f86-943f-ce753bc61a71)

![image](https://github.com/user-attachments/assets/5ca23d46-1644-4d69-a964-ca05ea8372bf)
